vignettes/preprocess.Rmd

```diff
@@ -204,18 +204,19 @@ Here, `var3`, `var4`, and `var5` all have no variability, so these variables are
 preprocess_data(dataset = nonvar_df, outcome_colname = "outcome")
 ```
 
-You can read the [`caret::preProcess`](https://www.rdocumentation.org/packages/caret/versions/6.0-86/topics/preProcess) documentation for more information. If you want to include them, you can use the argument `remove_nzv=FALSE`. For this to work, you cannot collapse correlated features (otherwise it errors out because of the underlying `caret` function we use). 
+You can read the [`caret::preProcess`](https://rdrr.io/cran/caret/man/preProcess.html) documentation for more information. If you want to include them, you can use the arguments `remove_nzv=FALSE` and `remove_zv=FALSE`. For this to work, you cannot collapse correlated features (otherwise it errors out because of the underlying `caret` function we use). 
```

Comment:
pkgdown automatically links to package documentation on rdrr.io if you use this syntax: `caret::preProcess()`

Reply:
Oh yeah forgot about that!